interp/src/debugger/debugger_core.rs

```diff
@@ -175,6 +198,28 @@ impl<C: AsRef<Context> + Clone> Debugger<C> {
             .collect_vec();
         self.manipulate_breakpoint(Command::Delete(parsed_bp_ids));
     }
+
+    pub fn cont(&mut self) -> StoppedReason {
+        self.is_running = true;
+        let _ = self.do_continue(); //need to error handle
+        let bps = self
+            .debugging_context
+            .hit_breakpoints()
+            .map(|x| self.grp_idx_to_name(x))
+            .collect_vec();
+        if self.interpreter.is_done() {
+            StoppedReason::Done
+        } else if !bps.is_empty() {
+            StoppedReason::Breakpoint(bps)
+        } else {
+            StoppedReason::PauseReq
```

Comment:
I don't think this stop reason is possible. `do_continue` only exits when the
program has finished running or has hit a breakpoint. To my knowledge, there is
no actual way to preempt it. That is, it isn't possible to cause the debugger to
exit the continue loop while it is running in the absence of one of the
termination conditions. Doing so would require some fancy threading work which
doesn't exist at the moment